easybuild/easyconfigs/g/Genome_Profiler/Genome_Profiler-2.1-foss-2016b-Perl-5.24.0.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CmdCp'
+
+name = 'Genome_Profiler'
+version = '2.1'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://sourceforge.net/projects/genomeprofiler/'
+description = """Genome Profiler (GeP) is a program to perform whole-genome multilocus
+ sequence typing (wgMLST) analysis for bacterial isolates"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://sourceforge.net/projects/genomeprofiler/files/Genome_Profiler_%(version)s/']
+sources = [{'download_filename': 'GeP.pl', 'filename': 'GeP-%(version)s.pl'}]
+checksums = ['212624965f84316663499751b117608e30b2074675c0e8d38aa8c5f8f6a86a99']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+]
+
+# skip uncompress because this is plain .pl file
+skipsteps = ['source']
+
+cmds_map = [("GeP*.pl", "cp %(source)s GeP.pl")]
+
+files_to_copy = [(['GeP.pl'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/GeP.pl'],
+    'dirs': [],
+}
+
+postinstallcmds = [
+    "sed -i -e 's|/usr/bin/perl|/usr/bin/env\ perl|' %(installdir)s/bin/GeP.pl",  # fix shebang line
+    "chmod +x %(installdir)s/bin/GeP.pl",
+]
+
+modloadmsg = "To launch execute script GeP.pl\n\r"
```

Comment:
@pescobar Why the `\r`, the test suite doesn't like that...

Reply:
@boegel because of this https://github.com/easybuilders/easybuild-framework/issues/2405

Reply:
OK, but then we should fix that bug rather than using dirty tricks to work around it. ;)